go/processor/floria/processor.go

```diff
@@ -108,6 +112,27 @@ func (p *processor) Run(
 	}, nil
 }
 
+func setUpAccessList(transaction tosca.Transaction, context tosca.TransactionContext, revision tosca.Revision) {
+	if transaction.AccessList == nil {
+		return
+	}
+
+	if transaction.Recipient != nil {
+		context.AccessStorage(*transaction.Recipient, tosca.Key{})
```

Comment:
Should this use [`AccessAccount`](https://github.com/Fantom-foundation/Tosca/blob/cd54781393a81426f6174a67d8a6b08b95677a9d/go/tosca/interpreter.go#L95) instead?